sdk/remove.js

```diff
@@ -18,17 +17,17 @@ const remove = async function remove(funcName, apiKey) {
     resolveWithFullResponse: true,
     simple: false,
   };
-  let response;
   try {
-    response = await rp.delete(options);
+    const response = await tryRequest(rp.delete(options));
+    return { status: response.statusCode, body: response.body };
   } catch (err) {
-    throw new Error(translateErrorCode('ERR_NO_BACKEND'));
-  }
-  if (response.statusCode < 200 || response.statusCode >= 300) {
-    if (get(response, 'body.errorCode')) {
-      throw new Error(translateErrorCode(response.body.errorCode));
+    // console.log(JSON.stringify(err));
+    if (err instanceof HTTPError) {
+      console.log('http error');
```

Comment:
why?!?

and more importantly why create a special case in the first place?

Reply:
This is the way it was implemented by the author of our `sdk/deploy`and I was just trying to maintain uniformity.